### PR TITLE
docs: add docs to download data with pg_dump

### PR DESCRIPTION
**Pull request by AntBush**
Docs for dumping data with pg_dump

### CODE REVIEW

File: docs/Data_Dump_With_Pg_Dump.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Creating a data dump with pg_dump
+
+## Connecting to pod
+
+Refer to the "Selecting leader pod in OpenShift" in [Process to Manipulate Data](./process_to_manipulate_data.md)
+
+## Creating and downloading the dump
+
+Once you've identified to the leading pod, you'll be in the position to dump the data. Using `kubectl exec -it <leading pod> -- /usr/pgsql-14/bin/pg_dump ccbc > backup.sql` will dump both the data and the schema to your local machine with the filename "backup.sql". To only download the data without the schema add `--data-only` after ccbc.
```
Comment (marcellmueller) [inline docs/Data_Dump_With_Pg_Dump.md:9]: `kubectl exec -it <leading pod> -- pg_dump ccbc > backup.sql`

Seems like you don't actually need to link directly to the executable

Comment (AntBush) [reply docs/Data_Dump_With_Pg_Dump.md:9]: Oh nice! I'll fix that up

**Review by marcellmueller — APPROVED** [review]
Nice work!

**Review by VB-at-Bis — APPROVED** [review]
Perfect

**Pull request merged**